src/components/search/results/resultsList/ResultItem/ResultItem.tsx

```diff
@@ -153,6 +158,12 @@ export const ResultItem: FC<ResultItemProps> = observer(({ hit, url, index }) =>
                                 <Grid item component="span" className={classes.collection}>
                                     {collection}
                                 </Grid>
+                                {hit._dedup_hide_result && (
+                                    <Grid item component="span" className={classes.collection} ml={2}>
+                                        {reactIcons.fileCopy} {t('duplicate', 'Duplicate')}: [
```

Comment:
is it a number of duplicates? if so maybe name it "Duplicates"

Reply:
Duplicates is better here indeed, thanks!